src/convertToObject.js

```diff
@@ -1,12 +1,23 @@
 'use strict';
 
-/**
+/*
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleUpgrated = sourceString
+    .split(';')
+    .map((el) => el.split(':'))
+    .reduce((prev, oneStyle) => {
+      if (oneStyle[0] && oneStyle[1]) {
+        return { ...prev, [oneStyle[0].trim()]: oneStyle[1].trim() };
```

Comment:
[CODE STYLE] - Avoid variable namings like `styleUpgrated`. It contains a typo and doesn't accurately describe the contents. Consider a name that reflects the purpose of the variable, such as `styleObject`.

Reply:
good point 🔥 
don't do it
```suggestion
        prev[property.trim()] = value.trim();
```